src/internal/modules/account/helpers.js

```diff
@@ -0,0 +1,33 @@
+const { throwIfError } = require('@envage/hapi-pg-rest-api');
+const internalConfig = require('internal/config');
+const services = require('internal/lib/connectors/services');
+
+const getEmailRegex = (config = internalConfig) => {
+  return (config.isLocal || config.testMode)
+    ? /(\.gov\.uk|gmail\.com)$/
+    : /\.gov\.uk$/;
+};
+
+const getUserById = async (userId) => {
+  const { data: user, error } = await services.idm.users.findOne(userId);
+  throwIfError(error);
+  return user;
+};
+
+const getUserByEmail = async (email, config = internalConfig) => {
```

Comment:
I think there may already be a `findOneByEmail` method on `services.idm.users` connector